src/env-inl.h

```diff
@@ -727,9 +727,9 @@ void Environment::SetUnrefImmediate(Fn&& cb) {
 }
 
 template <typename Fn>
-void Environment::SetImmediateThreadsafe(Fn&& cb) {
+void Environment::SetImmediateThreadsafe(Fn&& cb, bool refed) {
```

Comment:
Would prefer an enum rather than a magical boolean :-)

Reply:
@jasnell Hm yeah … this reaches into the `CallbackQueue` implementation which also takes this as a boolean argument and returns the result as a boolean, so maybe in a follow-up PR? I’d like to be consistent here…